src/Core/Domain/Carrier/Command/AddCarrierCommand.php

```diff
@@ -41,7 +41,12 @@ public function __construct(
         private string $trackingUrl,
         private int $position,
         private bool $active,
-        private ?string $logoPathName,
+        private bool $isShippingHandling,
+        private bool $isFree,
+        private int $shippingMethod,
+        private int $idTaxRuleGroup,
+        private bool $rangeBehavior,
+        private ?string $logoPathName
```

Comment:
```suggestion
        private ?string $logoPathName = null
```
Maybe this one could be optional since it's nullable anyway?